inappmessaging/USERGUIDE.md

```diff
@@ -347,9 +347,11 @@ Documents targeting Product Managers:
 
 ## <a name="changelog"></a> Changelog
 
-### 5.0.0 (in-progress)
+### 5.0.0 (2021-09-10)
```

Comment:
What about delay between campaigns? Was it not a part of 5.0.0?

Reply:
no, it is not included in this release